pkg/pipeline/pipeline.go

```diff
@@ -37,3 +44,23 @@ func GetAllPipelineNames(p cli.Params) ([]string, error) {
 	}
 	return ret, nil
 }
+
+func List(c *cli.Clients, opts metav1.ListOptions, ns string) (*v1beta1.PipelineList, error) {
+
+	pipelineGroupResource := schema.GroupVersionResource{Group: "tekton.dev", Resource: "pipelines"}
+	unstructuredTR, err := plist.AllObjecs(pipelineGroupResource, c, ns, opts)
```

Comment:
Any reason for calling this `unstructuredTR`? Maybe change to `P` or `Ps` for pipelines?

Reply:
Good catch it's my bad, copy-paste err